app/views/members/_account.html.haml

```diff
@@ -9,3 +9,17 @@
   = member.account_type
   account
 
+%p
+<<<<<<< HEAD
```

Comment:
You've committed a git conflict marker (the `<<< === >>>` stuff). When you merge or rebase and git tells you that it couldn't resolve a conflict, you need to resolve these conflicts manually before continuing with the merge/rebase. Git doesn't care what you put in your files, but conflict markers are not syntactically correct HAML or Ruby code :-) I've written a [slightly more detailed explanation of conflict-resolution in Git here](http://wiki.growstuff.org/index.php/Git#Merging_in_Git) - could it be made clearer?